src/www/webpack_cordova.mjs

```diff
@@ -0,0 +1,69 @@
+// Copyright 2020 The Outline Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+import path from "path";
+import CopyPlugin from "copy-webpack-plugin";
+import HtmlWebpackPlugin from "html-webpack-plugin";
+import webpack from "webpack";
+import {baseConfig, browserConfig, require, __dirname, GENERATE_CSS_RTL_LOADER} from "./webpack_base.mjs";
+import {merge} from "webpack-merge";
+
+const BABEL_LOADER = {
+  loader: "babel-loader",
+  options: {
+    presets: ["@babel/preset-env"],
+  },
+};
+
+export default merge(baseConfig, browserConfig, {
```

Comment:
webpack merge is a more flexible version of what we were doing before and allowed me to easily split up the configs in this way.